### PR TITLE
Final Milestone : Forex Metrics App

### PR DESCRIPTION
**Pull request by zdnahom**
## Final Milestone: Forex Metrics App
In this milestone I : 
- [x] Implemented Homepage
- [x] Implemented Detail page
- [x] Made routes dynamic
- [x] Did a test for all components
- [x] Followed the [design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))

### CODE REVIEW

File: src/__tests__/CurrencyPair.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      changes: -0.06200000000001182,
+      date: '2023-04-18 18:48:20',
+    };
+    const { container } = render(
+      <BrowserRouter>
+        <CurrencyPair exchangeItem={exchangeData} />
+      </BrowserRouter>,
+    );
+    expect(container).toMatchSnapshot();
+  });
```
Comment (TadesseDev) [inline src/__tests__/CurrencyPair.test.js:25]: - [ ] Dear @zdnahom, I notice you have added only `Snapshot` tests to your project. Please remember it's mandatory to test out several components of the app and some pure functions in Redux code. Also please consider adding integration tests using React Testing Library.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @zdnahom 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Good commit history  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
- [x] Dear @zdnahom, I like your video presentation, But the thing is it missed one important feature, which is the highlight of a piece of the code you find interesting or you are proud of making. This is important as it would help us practice presenting our project professionally in a fixed timeline. 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/CurrencyPair.test.js (hunk 1 above)
Comment (zdnahom) [reply src/__tests__/CurrencyPair.test.js:25]: Thanks for your review @TadesseDev, I will add more tests👍

Comment (TadesseDev) [reply src/__tests__/CurrencyPair.test.js:25]: You are doing great @zdnahom. 🙏

File: src/components/CurrencyPair.js [hunk 2]
```diff
@@ -1,4 +1,49 @@
...
+const CurrencyPair = ({ exchangeItem, handleSelect }) => {
+  const modifiedTicker = exchangeItem.ticker.split('/').join('_');
+  return (
+    <Link style={{ textDecoration: 'none' }} key={exchangeItem.ticker} to={`/${modifiedTicker}`}>
+      <li className="currency-card">
+        <button type="button" onClick={() => handleSelect(exchangeItem.ticker)}>
+          <FontAwesomeIcon icon={faCircleArrowRight} className="forward-icon" />
+          <div className="currency-card-content">
+            <div className="currency-rate">
+              Bid
```
[the remark this answers is not in the document]
Comment (zdnahom) [reply src/components/CurrencyPair.js:16]: I will fix the overflow and decrease the font size, Thanks @TadesseDev 🙌

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @zdnahom, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**